tests/unittests/net/artifacts/photon_net_config/etc/systemd/network/10-cloud-init-vlan102.network

```diff
@@ -0,0 +1,17 @@
+[Address]
+Address=10.100.102.10/24
+
+[Link]
+MTUBytes=1400
+
+[Match]
+Name=vlan102
+
+[Network]
+DHCP=no
+DNS=8.8.8.8 1.0.0.1
+Domains=vlan102.local
+
+[Route]
+Destination=0.0.0.0/0
+Gateway=10.100.102.1
```

Comment:
Same comments here as on `10-cloud-init-vlan101.network`. There should be a `Table` key and a `RoutingPolicyRule` section.